core/iwasm/common/wasm_runtime_common.c

```diff
@@ -315,7 +319,20 @@ wasm_runtime_get_exec_env_tls()
 {
     return exec_env_tls;
 }
-#endif /* end of OS_ENABLE_HW_BOUND_CHECK */
+#endif
+
+#ifdef OS_ENABLE_BLOCK_INSN_INTERRUPT
+static void
+interrupt_block_insn_sig_handler(int sig)
+{
+    bh_assert(sig == SIGUSR1);
```

Comment:
Should not use SIGUSR1 in non platform related code, if to assert it, suggest a the signal handler in posix_thread.c and assert there, and call this function inside that signal handler.